install/kubernetes/cilium/templates/cilium-agent/daemonset.yaml

```diff
@@ -712,13 +712,11 @@ spec:
           path: /sys/fs/bpf
           type: DirectoryOrCreate
       {{- end }}
-      {{- if .Values.cgroup.autoMount.enabled }}
```

Comment:
IIUC, apply-sysctl-overwrites is already guarded by .Values.cgroup.autoMount.enabled. Do you think anything else missing here ?


https://github.com/cilium/cilium/blob/master/install/kubernetes/cilium/templates/cilium-agent/daemonset.yaml#L425-L516

Reply:
Yep, but if you disable ```cgroup.autoMount```, hostproc volume wouldn't exist => daemonset wouldn't up at all.
My OS already prepared ```cgroup-v2``` and I disabled cgroup autoMount step.
 
Another solution: is set ```bool``` options for ```apply-sysctl-overwrites```. Is it make sense?

Reply:
If you disable cgroup.autoMount, the mentioned initContainer apply-sysctl-overwrites will be disabled as per current behavior. 

Basically, I am confused what you want to achieve with this patch :thinking:

Reply:
oh sorry it already fixed in master... 
I've tested 1.12.0.

Reply:
thanks and looking forward to your contribution in any forms (e.g. pr, issues, discussion, etc)